gno.land/pkg/integration/testdata/use_both.txtar

```diff
@@ -0,0 +1,23 @@
+# load a package from the example packages directory.
+use gno.land/p/demo/ufmt
+use $WORK gno.land/r/importtest
```

Comment:
So you no longer need to do `gnokey addpkg ...`? 🤩

Reply:
yes, it should simplify some test :)